sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/ExpressionCompiler.java

```diff
@@ -564,6 +564,17 @@ A condition case (CASE WHEN p1 THEN v1 ... ELSE e END) has an odd number of
                         "_" + keyword + type1.nullableSuffix();
                 return new DBSPApplyExpression(node, functionName, type, ops.get(1));
             }
+            case LIKE:
+            case SIMILAR: {
```

Comment:
Is SIMILAR also standard SQL or from some DB-specific dialect?

Reply:
In fact SIMILAR doesn't work yet, it will require significant work.

Reply:
When it's ready I will post more documentation.